input/pagecontent/5_content-sharing.md

```diff
@@ -0,0 +1,78 @@
+
+Clients wishing to exchange structured information must:
+1. Handle FHIRcast events for anchor context types it supports: [FHIR resource]-[open\|update\|close\|select]
+2. Use a `[FHIR resource]-open` request to open a new resource which becomes the anchor context
+3. Make a `[FHIR resource]-update` request when appropriate. The `[FHIR resource]-update` request contains a `Bundle` resource which is a collection of resources that are atomically processed by the Hub with the anchor context's content being adjusted appropriately
+4. Maintain the current `versionId` of the anchor context provided by the Hub so that a subsequent `[FHIR resource]-update` request may provide the current `versionId` which will be validated by the Hub
+5. Appropriately process [FHIR resource]-[open\|update\|close\|select] events; note that a client may choose to ignore the contents of a `[FHIR resource]-update` event but should still track the `versionId` for subsequent use
+6. If a `[FHIR resource]-update` request fails with the Hub, the client may issue a [`GET Context`](3-16-get-context.html) request to the Hub in order to retrieve the current content in the anchor context and its current `versionId`
+7. When using websockets, clients will now receive the current content (if any exists) of the anchor context (if one has been established) in response to the Subscribe request, see [`return of current content`](#websocket-return-of-current-content).  Clients that don't support the exchange of structured information may ignore the content of the Subscribe response payload.
+
+### Example of Content Sharing in an Anchor Context
+The below example uses a `DiagnosticReport` as the anchor context.  However, the pattern of the example holds when other FHIR resource types are the anchor context.
+
+#### Diagnostic Report Content Sharing Basics
+When reporting applications integrate with PACS and/or RIS applications, a radiologist's (or other clinician's) workflow is centered on the final deliverable, a diagnostic report. In radiology, the imaging study (exam) is an integral resource with the report referencing one or more imaging studies. Structured data, many times represented by a FHIR `Observation` resource, may also be captured as part of a report.  In addition to basic context synchronization, a diagnostic report centered workflow builds upon the basic FHIRcast operations to support near real-time exchange of structured information between applications participating in a diagnostic report context.  Also, the `DiagnosticReport` resource contains certain attributes (such as report status), that are useful to PACS/RIS applications.  Participating applications may include clients such as reporting applications, PACS, EHRs, workflow orchestrators, and interactive AI applications.
```

Comment:
DxReport, and maybe Observation and other resources expected to be exchanged need profiles.

Reply:
Agreed, on my to do list.   Creating them is not a huge deal but I am trying to figure out how to have the IG Publisher to put them in a good place and not randomly in the ToC.  I see there is a CapabilityStatement profile (FSH) file with the defined profile ending up in the Artifacts tab.  I guess the same is fine and then I can also link the profiles' output files in the -update and get current context operations.